addons/mail/static/src/components/composer/composer.js

```diff
@@ -174,10 +174,22 @@ class Composer extends Component {
 
     /**
      * Post a message in the composer on related thread.
-     *
+     * Posting of the message could be aborted if it cannot be posted like if there are attachments
+     * currently uploading or if there is no text content and no attachments.
      * @private
      */
     async _postMessage() {
+        if (!this.composer.canPostMessage) {
+            if (this.composer.hasUploadingAttachment) {
+                this.env.services['notification'].notify({
+                    message: this.env._t("Please wait while the file is uploading."),
+                    sticky: true,
```

Comment:
I would not make it sticky. Generally:

- sticky notifications usually require a user interaction on the notification itself
- non-sticky notifications are feedback at a given point in time

It feels more natural as a non-sticky notifications.

Reply:
It should not be, I did it for a test (wanted to see the markup used) and forgot to remove it